### PR TITLE
Add `Skinners' Bulletin: July 2021` newspost

### PR DESCRIPTION
**Pull request by zeusminus**
To give a brief insight, this is a monthly bulletin project concerning skinning community and an attempt to revive the skin showcase newspost of ages ago, which @RockRoller01 and I have been working on for over a month with the help of the skinship staff. 

We're planning to have it released within today, if possible. Reviews are always welcome!

### CODE REVIEW

File: news/2021-09-14-skinners-bulletin-july-2021.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+In July, a total of **45** skins were published on the forum, **26** of which were made out of entirely original assets.
+
+![Original/Mixed Graph](/wiki/shared/news/2021-09-14-skinners-bulletin-july-2021/graph-skins.png)
+
+As for the creators of these skins, a total of **41** skinners contributed via their creations. Seeing the new faces forging the majority, we would like to welcome all **25** first-timers aboard!
+
+![New/Returning Graph](/wiki/shared/news/2021-09-14-skinners-bulletin-july-2021/graph-skinners.png)
+
+In place of the dessert, we would like to share some interesting albeit trivial facts with you: In this month, the highest number of skins released by a single person is **2**, and the number of skinners published that many skins are no more than **4**, leading up to average **1.1** skins per skinner overall.
```
Comment (RoanH) [inline news/2021-09-14-skinners-bulletin-july-2021.md:31]:
```suggestion
In place of the dessert, we would like to share some interesting albeit trivial facts with you: In this month, the highest number of skins released by a single person is **2**, and the number of skinners who published that many skins are no more than **4**, leading up to an average of **1.1** skins per skinner overall.
```

File: news/2021-09-14-skinners-bulletin-july-2021.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+### Arknights Ch'en v2.0
+
+Created with care by [Inflique](https://osu.ppy.sh/users/13865585), [Arknights Ch'en v2.0](https://osu.ppy.sh/community/forums/topics/1365860) ensures you a cyberpunk-esque gaming experience featuring Ch'en from the popular mobile game Arknights. Presenting you with a futuristic song selection panel, the skin makes you feel as if you're in a cockpit of a high-tech spaceship.
+
+Even though just a cursory look might be sufficient to impress many skin curators, the real deal with this skin is undoubtedly the details. Further enhancing the essence of its theme, each and every corner of Arknights Ch'en is full of flavour texts, such as the alert message found on the results screen or the faction logo put in place of the game mode selection button. Moreover, the menu sounds carry out the futuristic theme and contain the character voice lines of Ch'en, maintaining the feeling of playing Arknights by clicking the circles. Alongside everything else, the colour choices fit the environment perfectly, ensures you a pleasant gaming session inside an exosuit jumped out of a typical sci-fi movie.
```
Comment (RoanH) [inline news/2021-09-14-skinners-bulletin-july-2021.md:61]:
```suggestion
Even though just a cursory look might be sufficient to impress many skin curators, the real deal with this skin is undoubtedly the details. Further enhancing the essence of its theme, each and every corner of Arknights Ch'en is full of flavour texts, such as the alert message found on the results screen or the faction logo put in place of the game mode selection button. Moreover, the menu sounds carry out the futuristic theme and contain the character voice lines of Ch'en, maintaining the feeling of playing Arknights by clicking the circles. Alongside everything else, the colour choices fit the environment perfectly, ensuring you a pleasant gaming session inside an exosuit jumped out of a typical sci-fi movie.
```

File: news/2021-09-14-skinners-bulletin-july-2021.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+Last but not least, Raiden Shogun - Baal is one of the perfect examples of how character-orientation and minimalism should be mixed into one. Nonetheless, we are but the messenger, and deciding on the skin that goes well with your taste falls onto your hands, so give its showcase video a go before making a decision:
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/FhY1ikQljiU" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+And that wraps up our skin showcases for July 2021! We'll come back with more of them in the forthcoming issues, but until then, have a look around the rest of the bulletin to enlighten yourself with everything skinning.
+
+## News & Tips
+
+Curious about what is going on with the skinning community? Read all about it right away!
```
Comment (RoanH) [inline news/2021-09-14-skinners-bulletin-july-2021.md:85]:
```suggestion
Curious about what is going on within the skinning community? Read all about it right away!
```

File: news/2021-09-14-skinners-bulletin-july-2021.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+### Completed Skins Compendium Rework
+
+Previously consisting of a long list of skins grouped under 5 prominent categories, the [Completed Skins Compendium](https://osu.ppy.sh/community/forums/topics/686664) has been pulled out of forums and replaced by [a fully operational website](https://compendium.skinship.xyz/ "skinship") on its own. Thanks to its advanced sorting options coupled with the search bar, the new compendium ensures that none of the skins require browsing further than skin-deep!
+
+Although the website is being hosted outside of osu!, the content is based on the [Completed Skins subforum](https://osu.ppy.sh/community/forums/109), meaning that the skins should be wholly original and safe-for-work as per the [rules of the Skinning forum](https://osu.ppy.sh/community/forums/topics/318193).
```
Comment (RoanH) [inline news/2021-09-14-skinners-bulletin-july-2021.md:91]: Should be sounds kind of uncertain, while any skin that doesn't follow the forum rules also won't make it into the compendium. Unless the rules part only applies to the NSFW statement and you want to imply that skins might not be wholly original.

```suggestion
Although the website is being hosted outside of osu!, the content is based on the [Completed Skins subforum](https://osu.ppy.sh/community/forums/109), meaning that the skins are wholly original and safe-for-work as per the [rules of the Skinning forum](https://osu.ppy.sh/community/forums/topics/318193).
```

**Review by RoanH — DISMISSED** [review]
The videos are still set to private, I assume that should be changed before this is merged.

Left 4 other small suggestions, though feel free to ignore them.

**Comment by RockRoller01** [comment]
> The videos are still set to private
This is on purpose to prevent people from spoiling everything, the moment this gets merged they will be set to public.

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**